3.24 release blog (#2361)
sourcegraph/about: blogposts/2021/release-3.24.md

---
title: "Sourcegraph 3.24 release"
publishDate: 2021-01-20T10:00-07:00
description: "Sourcegraph 3.24 is released."
tags: [blog, release]
slug: "release/3.24"
published: true
changelogItems:

>   - description: Added panels to the Sourcegraph monitoring dashboards

Reviewer: This one feels a bit vague, I think if we can't fit the details in the changelog (new dashboards reference, updated panels, version/alert annotations) it would be best to just omit this entry 😄 (speaking as the person who wrote this entry in the changelog)

Reviewer 2: @aileenrose are we limited to a certain number of characters for this post type? 

@bobheadxi do you have a suggestion for the description that shares more of the details? If we don't have an imposed word limit, we can share whatever is the most valuable, helpful, and informative for the user and reader of the post.

Reviewer: ```suggestion
  - description: Monitoring dashboards now include links to relevant documentation and annotation overlays for alerts and version changes.
```

    url: https://docs.sourcegraph.com/admin/observability/metrics#grafana
    category: Admin
  - description: Suggested filters in the search results page can now be scrolled.
    url: https://github.com/sourcegraph/sourcegraph/pull/17097
    category: Search

>   - description: Dashboard links now included in monitoring alerts.

Reviewer: ```suggestion
  - description: Dashboard links included in monitoring alerts are now more specific.
```

Reviewer: The links have existed for a while, the change is to make them more specific and helpful

Reviewer 2: @bobheadxi do you have more any more detail on what makes the dashboard links more specific and helpful?

Reviewer: From the changelog:

Dashboard links included in monitoring alerts now:
- link directly to the relevant Grafana panel, instead of just the service dashboard. #17014
- link to a time frame relevant to the alert, instead of just the past few hours. #17034

I think this entry could point out both, e.g. "Links now lead directly to the relevant panel and time frame"

Reviewer: ```suggestion
  - description: Dashboard links included in monitoring alerts now point directly to the relevant panel and time frame.
```
